frontend/testing/playwright/pages/DashboardPage.ts

```diff
@@ -15,6 +16,70 @@ export class DashboardPage extends BasePage {
   }
 
   public async clickOnCreateAppLink(): Promise<void> {
-    await this.page.getByRole('link', { name: this.textMock('dashboard.new_service') }).click();
+    await this.getLinkByTextKey('dashboard.new_service').click();
+  }
+
+  public async checkThatThereIsNoFavouriteAppInList(appName: string): Promise<void> {
+    // The .first() is added becuase the key is used two places; one in favourite list, and one in all applications list
+    await this.getMenuItemByTextKey('dashboard.unstar', { appName }).first().isHidden();
+  }
+
+  public async clickOnFavouriteApplication(appName: string): Promise<void> {
+    await this.getMenuItemByTextKey('dashboard.star', { appName }).click();
```

Comment:
Ref. prev comment, here I do not know if the `getMenuItemByTextKey` is implemented using getByRole or something else. I find it easier to read and maintain by avoid wrapper methods for this scenario. 😊